analyzers/tests/SonarAnalyzer.UnitTest/Rules/Hotspots/UsingCookiesTest.cs

```diff
@@ -36,45 +37,76 @@ namespace SonarAnalyzer.UnitTest.Rules
     [TestClass]
     public class UsingCookies
     {
+        private readonly VerifierBuilder builderCS = new VerifierBuilder()
+            .AddAnalyzer(() => new CS.UsingCookies(AnalyzerConfiguration.AlwaysEnabled))
+            .WithBasePath(@"Hotspots\");
+
+        private readonly VerifierBuilder builderVB = new VerifierBuilder()
+            .AddAnalyzer(() => new VB.UsingCookies(AnalyzerConfiguration.AlwaysEnabled))
+            .WithBasePath(@"Hotspots\");
+
 #if NETFRAMEWORK // HttpCookie is available only when targeting .Net Framework
         [TestMethod]
         public void UsingCookies_CS_Net46() =>
-            OldVerifier.VerifyAnalyzer(
-                @"TestCases\Hotspots\UsingCookies_Net46.cs",
-                new CS.UsingCookies(AnalyzerConfiguration.AlwaysEnabled),
-                GetAdditionalReferencesForNet46());
+            builderCS
+                .AddPaths("UsingCookies_Net46.cs")
+                .AddReferences(GetAdditionalReferencesForNet46())
+                .Verify();
 
         [TestMethod]
         public void UsingCookies_VB_Net46() =>
-            OldVerifier.VerifyAnalyzer(
-                @"TestCases\Hotspots\UsingCookies_Net46.vb",
-                new VB.UsingCookies(AnalyzerConfiguration.AlwaysEnabled),
-                GetAdditionalReferencesForNet46());
+            builderVB
+                .AddPaths("UsingCookies_Net46.vb")
+                .AddReferences(GetAdditionalReferencesForNet46())
+                .Verify();
 
         internal static IEnumerable<MetadataReference> GetAdditionalReferencesForNet46() =>
             FrameworkMetadataReference.SystemWeb;
 
 #else
         [TestMethod]
         public void UsingCookies_CS_NetCore() =>
-            OldVerifier.VerifyAnalyzer(
-                @"TestCases\Hotspots\UsingCookies_NetCore.cs",
-                new CS.UsingCookies(AnalyzerConfiguration.AlwaysEnabled),
-                GetAdditionalReferencesForNetCore(Constants.DotNetCore220Version));
+            builderCS
```

Comment:
I think we should let only this test with AspNet Core 2 and remove the other tests with Asp.Net Core 2.